### PR TITLE
Add manual installation steps for interceptor bridge installation 

### PR DESCRIPTION
**Pull request by saialekhya-postman**
Add manual installation steps for interceptor bridge installation for Mac, Linux, Windows.

### CODE REVIEW

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 1]
```diff
@@ -162,6 +162,31 @@ If the aforementioned troubleshooting steps do not fix the problem, you may need
 
 > If any dependencies cannot be removed due to permission issues, you’ll be prompted to delete them manually.
 
+ If you are unable to resolve ``INTERNET_CONNECTIVITY`` error due to proxy issues and want to use the integration here are the manual installation steps:
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/interceptor.md:165]: Looks like there's an extra space at the beginning here

Comment (claireriley) [reply src/pages/docs/postman/sending-api-requests/interceptor.md:165]: Would the following work?

>  If you are unable to resolve an ``INTERNET_CONNECTIVITY`` error, you can manually install Interceptor:

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 2]
```diff
@@ -162,6 +162,31 @@ If the aforementioned troubleshooting steps do not fix the problem, you may need
...
+
+**1. MacOS/Linux**:
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/interceptor.md:167]: Let's change this to bullet points and indent the details below

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 3]
```diff
@@ -162,6 +162,31 @@ If the aforementioned troubleshooting steps do not fix the problem, you may need
...
+
+* **[MacOS-only]**Node(>v6.0.0) should be installed and available at `/usr/local/node` or `/usr/local/bin/node`, or you have the environment variable `NVM_BIN` set.
+* Install bridge for [MacOS](https://go.pstmn.io/interceptor-bridge-macos)/ [Linux](https://go.pstmn.io/interceptor-bridge-linux)
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/interceptor.md:170]: Let's remove the space before linux

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 4]
```diff
@@ -162,6 +162,31 @@ If the aforementioned troubleshooting steps do not fix the problem, you may need
...
+* Run the script (install_host.sh)
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/interceptor.md:171]: Would we want install_host.sh to be styled `install_host.sh`?

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 5]
```diff
@@ -162,6 +162,31 @@ If the aforementioned troubleshooting steps do not fix the problem, you may need
...
+* After executing the script check the following:
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/interceptor.md:172]: let's add "items" to the end of this and add a comma after "script"

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 6]
```diff
@@ -162,6 +162,31 @@ If the aforementioned troubleshooting steps do not fix the problem, you may need
...
+  1. Verify  `InterceptorBridge` file exists in this path ``$HOME/.postman``.
+  2. Verify in this path `~/Library/ApplicationSupport/Google/Chrome/NativeMessagingHosts` for MacOs or `~/.config/google-chrome/NativeMessagingHosts` for Linux for the presence of manifest file ``com.postman.postmanapp.json`` with the correct extension ID ``aicmkgpgakddgnaphhhpliifpcfhicfo``, and with correct path ``$HOME/.postman`` to the executable file(InterceptorBridge).
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/interceptor.md:174]: I tried splitting it into two sentences below to make it a bit easier to read, but either version is okay. I would format ``InterceptorBridge`` though as it looks like we did earlier on in the doc, and we can remove the parenthesis.

> Verify the manifest file ``com.postman.postmanapp.json`` is present at `~/Library/ApplicationSupport/Google/Chrome/NativeMessagingHosts` for MacOs or `~/.config/google-chrome/NativeMessagingHosts` for Linux. Check that it has the correct extension ID ``aicmkgpgakddgnaphhhpliifpcfhicfo``, and the correct path ``$HOME/.postman`` to the executable file ``InterceptorBridge``.

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 7]
```diff
@@ -162,6 +162,31 @@ If the aforementioned troubleshooting steps do not fix the problem, you may need
...
+
+**2. Windows**:
+
+* [Install bridge](https://go.pstmn.io/interceptor-bridge-windows)
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/interceptor.md:178]: We may want to add periods after this and the following point to be consistent with the one before it.

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 8]
```diff
@@ -162,6 +162,31 @@ If the aforementioned troubleshooting steps do not fix the problem, you may need
...
+* Run the script (install_host.bat)
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/interceptor.md:179]: Would we want to stylize "install_host.bat" as well?

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 9]
```diff
@@ -162,6 +162,31 @@ If the aforementioned troubleshooting steps do not fix the problem, you may need
...
+* After executing the script check the following:
+  1. Verify  `InterceptorBridge.exe` file exists in this path ``%USERPROFILE%/.postman/``.
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/interceptor.md:181]: add a "the" or "an" after "Verify"

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 10]
```diff
@@ -162,6 +162,31 @@ If the aforementioned troubleshooting steps do not fix the problem, you may need
...
+  2. Verify in this path `HKEY_CURRENT_USER\SOFTWARE\Google\Chrome\NativeMessagingHosts` for existence of manifest file ``com.postman.postmanapp.json`` with the correct extension ID ``aicmkgpgakddgnaphhhpliifpcfhicfo`` and correct path ``%USERPROFILE%/.postman/`` to the executable file(InterceptorBridge.exe).
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/interceptor.md:182]: How about

> Verify in this path `HKEY_CURRENT_USER\SOFTWARE\Google\Chrome\NativeMessagingHosts` that the manifest file ``com.postman.postmanapp.json`` exists with the correct extension ID ``aicmkgpgakddgnaphhhpliifpcfhicfo`` and correct path ``%USERPROFILE%/.postman/`` to the executable file ``InterceptorBridge.exe``.

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 11]
```diff
@@ -162,6 +162,31 @@ If the aforementioned troubleshooting steps do not fix the problem, you may need
...
+
+**Note:**
+
+ 1. To run the script depending on your OS, you might need to double-click or execute the installer script via a shell. Users on MacOS/Windows might see a security warning. For example, to override the security on MacOS, you may need to right click > open.
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/interceptor.md:186]: Let's change the "might"s to "may"s

File: src/pages/docs/postman/sending-api-requests/interceptor.md [hunk 12]
```diff
@@ -162,6 +162,31 @@ If the aforementioned troubleshooting steps do not fix the problem, you may need
...
+
+ 2. If you have installed Postman app via snap(in LINUX machines), Interceptor  integration won't get connected with this manual steps also. In such case please install Postman app outside of snap.
```
Comment (claireriley) [inline src/pages/docs/postman/sending-api-requests/interceptor.md:188]: Maybe the following?

> If you have installed Postman app via snap (in LINUX machines), Interceptor will not connect and you must reinstall the Postman app outside of snap.

File: src/pages/docs/postman/sending-api-requests/interceptor.md (hunk 4 above)
Comment (saialekhya-postman) [reply src/pages/docs/postman/sending-api-requests/interceptor.md:171]: no need, let it be like a link

**Pull request merged**